code/modules/mob/living/carbon/carbon.dm

```diff
@@ -291,42 +291,47 @@
 
 /mob/living/carbon/proc/check_self_for_injuries()
 	var/mob/living/carbon/human/H = src
-	visible_message( \
-		text("<span class='notice'>[src] examines [].</span>",gender==MALE?"himself":"herself"), \
+	visible_message("<span class='notice'>[src] examines [H.p_them()]selves.</span>", \
 		"<span class='notice'>You check yourself for injuries.</span>" \
 		)
+	var/list/status_list = list()
 
 	var/list/missing = list("head", "chest", "groin", "l_arm", "r_arm", "l_hand", "r_hand", "l_leg", "r_leg", "l_foot", "r_foot")
 	for(var/X in H.bodyparts)
 		var/obj/item/organ/external/LB = X
 		missing -= LB.limb_name
-		var/status = ""
+		var/status
 		var/brutedamage = LB.brute_dam
 		var/burndamage = LB.burn_dam
 
-		if(brutedamage > 0)
-			status = "bruised"
-		if(brutedamage > 20)
-			status = "battered"
-		if(brutedamage > 40)
-			status = "mangled"
+		switch(brutedamage)
+			if(0.1 to 20)
+				status = "bruised"
+			if(20 to 40)
+				status = "bruised"
```

Comment:
The old second line was "battered" not "bruised" a second time

Reply:
Good spot. Cheers.